DeepSpeech.py

```diff
@@ -1564,7 +1564,7 @@ def end(self, session):
 
                     # if the job was for validation dataset then append it to the COORD's _loss for early stop verification
                     if (job.set_name == 'dev') and (FLAGS.early_stop is True):
-                        COORD._dev_losses.append(job.loss)
+                        COORD._dev_losses[COORD._epoch-1].append(job.loss)
```

Comment:
Yes, this was already wrong in the former PR for this problem. I've overseen it in the first one's review.
However: This fix is too complicated.
The loss is persisted in each validating epoch - look for 
```
self.loss = agg_loss / num_jobs
```
I think this can be used for the early stopping.